dependency-check-suppressions.xml

```diff
@@ -186,5 +186,7 @@
         <packageUrl regex="true">^pkg:maven/org\.elasticsearch/elasticsearch@.*$</packageUrl>
         <vulnerabilityName>CVE-2019-7611</vulnerabilityName>
         <vulnerabilityName>CVE-2018-17244</vulnerabilityName>
+        <!-- Temporary suppression to ignore until it's fixed in master -->
+        <vulnerabilityName>CVE-2019-7614</vulnerabilityName>
```

Comment:
Did you open a ticket for the actual fix of this CVE? Or, is there one already?

Reply:
There is one already RDM-5664. It's in ReadyForTest